src/EntityFramework.SqlServer/SqlProviderServices.cs

```diff
@@ -132,6 +133,16 @@ public static bool TruncateDecimalsToScale
         }
 
         /// <summary>
+        /// Set this flag to false to prevent ordering using a row_number() function for queries with OFFSET operation, and order by columns directly instead.
```

Comment:
Consider switching to the same organization I used in https://github.com/aspnet/EntityFramework6/pull/182/files#diff-4b07700a7952daedde75b9c4de664a41L134. It would probably go along the lines of:

```
<summary>
Gets or sets a value indicating whether the ROW_NUMBER() function is used in sort expression passed to the ORDER BY clause when OFFSET is present in query. The default value of <c>true</c> is recommended to obtain query results that are stable for paging operations. The value of <c>false</c> can be used for compatibility with previous versions of EF and will cause the sort expression to be passed unmodified to the ORDER BY clause, which can lead to unstable results if the ordering is ambiguous. 
</summary>
<remarks>This flag only applies to SQL Server 2012 or later. This flag does not affect queries that have already been translated to SQL and cached, therefore applications that need to set the value to <c>false</c> for compatibility should do so before executing any queries.</remarks>
```
cc @rowanmiller @ajcvickers